Sources/OpenTelemetryInstrumentationSupport/SpanAttribute+RPCSemantics.swift

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Tracing open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Swift Tracing project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+#if swift(>=5.2)
+import TracingInstrumentation
+
```

Comment:
Also the namespace var:

```suggestion
extension SpanAttributes {
    /// Semantic conventions for RPC spans.
    public var rpc: RPCAttributes {
        get {
            .init(attributes: self)
        }
        set {
            self = newValue.attributes
        }
    }
}

```